privacy-security-explainer.md

```diff
@@ -0,0 +1,68 @@
+
+### User activation
+[User activation](https://html.spec.whatwg.org/multipage/interaction.html#activation) is defined within the HTML spec as an action the user can take which can result in certain types of HTML elements becoming activated. For example, a button becomes activated when a user clicks it with a mouse. The concept of user activation, differentiates user-caused events from injected events to prevent pages from spoofing user actions. User activation can be checked by the UA prior to session creation as well as during a session in response to appropriate `XRInputSource` events.
+
+### Implied consent
+A User Agent may use implied consent based, for example, on the install status of a web application or frequency and recency of visits. Given the sensitivity of XR data, caution is strongly advised when relying on implicit signals. 
+
+### Explicit consent
+It is often useful to get explicit consent from the user before exposing sensitive information. When gathering explicit user consent, User Agents must present an explanation of what is being requested and provide users the option to decline. Requests for user consent can be presented in many visual forms based on the features being protected and User Agent choice. While often associated with the [Permissions API](https://www.w3.org/TR/permissions/), the concept of user consent does not have exact overlap.
+
+### Timing of XR-related consent prompts
+**TODO** Fill this in with what was agreed up on in [#720](https://github.com/immersive-web/webxr/issues/720).
+
+### Duration of consent
+**TODO** Fill this in with what was agreed up on in [#721](https://github.com/immersive-web/webxr/issues/721).
+
+### Querying consent status
+**TODO** Fill this in with what is agreed upon in [#722](https://github.com/immersive-web/webxr/issues/722) and [#725](https://github.com/immersive-web/webxr/issues/725).
+
+## Data adjustments
+In some cases, security and privacy threats can be mitigated through throttling, quantizing, rounding, limiting, or otherwise adjusting the data reported from the WebXR APIs. This may sometimes be necessary to avoid fingerprinting, even in situations when user intent has been established.  However, data adjustment mitigations can only be used in situations which would not result in user discomfort.
+
+### Throttling
+Throttling is when sensitive data is reported at a lower frequency than otherwise possible. This mitigation has the potential to reduce a site's ability to infer user intent, location, or perform user profiling. However, when not used appropriately throttling runs a significant risk of causing user discomfort. In addition, under many circumstances it may be inadequate to provide a complete mitigation.  For example, 2D touch input data snooping has been proven possible at frequencies as [low as 20Hz](https://arxiv.org/pdf/1602.04115.pdf) via accelerometer data.
```

Comment:
This section feels like it could use an brief example illustrating how throttling could cause user discomfort, and maybe a counter example of a case where it's not a concern?

Reply:
probably should put these things in a todo so we can get this landed asap

Reply:
I'm fine with Manish's sentiment of landing ASAP.